ee/billing/test/test_quota_limiting.py

```diff
@@ -26,6 +28,66 @@ def setUp(self) -> None:
         super().setUp()
         self.redis_client = get_client()
 
+    @patch("posthoganalytics.feature_enabled", return_value=True)
+    @patch("posthoganalytics.capture")
+    def test_dont_quota_limit_feature_flag_enabled(self, patch_feature_enabled, patch_capture) -> None:
```

Comment:
These are actually in the wrong order, capture should come first then feature_enabled. See: https://wiki.dhruvs.space/programming/python/pytest/pytest-order-of-mock-and-parameterize/

Curious how this test is passing because you have `called_once_with` below... 🤔

Reply:
Think I need to run `patch_capture.assert_called_once_with` instead of doing the assert as I have it.